src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyClaimantClaimDiscontinuedNotificationHandlerTest.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.notification;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.handler.callback.BaseCallbackHandlerTest;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.notify.NotificationService;
+import uk.gov.hmcts.reform.civil.notify.NotificationsProperties;
+import uk.gov.hmcts.reform.civil.prd.model.Organisation;
+import uk.gov.hmcts.reform.civil.sampledata.CaseDataBuilder;
+import uk.gov.hmcts.reform.civil.service.OrganisationService;
+
+import java.util.Map;
+import java.util.Optional;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.handler.callback.camunda.notification.NotificationData.CLAIM_REFERENCE_NUMBER;
+import static uk.gov.hmcts.reform.civil.handler.callback.camunda.notification.NotificationData.LEGAL_ORG_NAME;
+
+@SpringBootTest(classes = {
+    NotifyClaimantClaimDiscontinuedNotificationHandler.class,
+    NotificationsProperties.class,
+    JacksonAutoConfiguration.class
+})
+class NotifyClaimantClaimDiscontinuedNotificationHandlerTest extends BaseCallbackHandlerTest {
+
+    public static final String TEMPLATE_ID = "template-id";
+
+    private static final String REFERENCE_NUMBER = "8372942374";
+
+    @MockBean
+    private NotificationService notificationService;
+
+    @MockBean
+    private NotificationsProperties notificationsProperties;
+
+    @MockBean
+    private OrganisationService organisationService;
+
+    @Autowired
+    private NotifyClaimantClaimDiscontinuedNotificationHandler handler;
+
+    @Nested
+    class AboutToSubmitCallback {
+
+        @BeforeEach
+        void setup() {
+            when(notificationsProperties.getNotifyClaimDiscontinuedLRTemplate()).thenReturn(
+                TEMPLATE_ID);
+        }
+
+        @Test
+        void shouldNotifyClaimantJudgmentByAdmission_whenInvoked() {
```

Comment:
shouldNotifyClaimantJudgmentByAdmission_whenInvoked the name looks wrong it should not be Judgment by admission

Reply:
Changed